pytest-testslide/setup.py

```diff
@@ -0,0 +1,50 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+
+import os
+
+from setuptools import setup  # type: ignore
+
+dir_path = os.path.dirname(os.path.realpath(__file__))
+
+ts_version = (
+    open(os.path.join(dir_path, os.pardir, "testslide/version")).read().rstrip()
```

Comment:
I believe this will fail to install later on, I recall doing something similar with TestSlide:

- `testslide/version` is **not** part of the built package.
- When another machine does `pip install pytest-teslide`, `setup.py` will fail to install due to this missing file.

I think the solution is:

- Add a `Makefile` rule to copy `testslide/version` to `pytest-testslide/version`.
- Add `pytest-testslide/version` to `pytest-testslide/MANIFEST.in`.